### PR TITLE
User story 5

### PR DESCRIPTION
**Pull request by krco253**
Affected Files:
src/test/cases/User_story_5/test.js
src/main/lib/course_portfolio_create.js
Package_lock.json
package.json
abet-submission-system/src/main/models/CoursePortfolio/index.js
abet-submission-system/src/test/cases/models/portfolio/portfolio.js

### CODE REVIEW

**Review by tyler-burkett — CHANGES_REQUESTED** [review]
The function for creating evaluation pdf's looks fine. We may need to address the format of PDF's in the future, though.
The function for generating a zip file looks good. Nice find on the external zip library.
The tests for the tasks don't actually have the instructor or course name in the objects in question. This needs to be there to verify they are in the database.

File: src/test/cases/models/portfolio/portfolio.js [hunk 1]
```diff
@@ -19,6 +19,35 @@ describe('Model - CoursePortfolio', () => {
...
+				.where('instructor_id', '1')
+			const slo = sl[0]
+			expect(slo).to.deep.equal({
+				id: 1,
+				course_id: 1,
+				instructor_id: 1,
+				semester_term_id: 1,
+				num_students: 5,
+				section: 1,
+				year: 2019
```
Comment (tyler-burkett) [inline src/test/cases/models/portfolio/portfolio.js]: Should have an instructor name

File: src/test/cases/models/portfolio/portfolio.js [hunk 2]
```diff
@@ -19,6 +19,35 @@ describe('Model - CoursePortfolio', () => {
...
+				.where('course_id', '1')
+			const slo = sl[0]
+			expect(slo).to.deep.equal({
+				id: 1,
+				course_id: 1,
+				instructor_id: 1,
+				semester_term_id: 1,
+				num_students: 5,
+				section: 1,
+				year: 2019
```
Comment (tyler-burkett) [inline src/test/cases/models/portfolio/portfolio.js]: Should have a course name

**Comment by krco253** [comment]
Updated affected files: 
/src/main/models/CoursePortfolio/index.js
/src/test/cases/models/portfolio/portfolio.js
/src/main/migrations/20191003180917.js
/src/dev/seeds/initial.js
/src/test/seeds/initial.js

**Review by tyler-burkett — APPROVED** [review]
New changes look perfect. Everything looks in order for the new database schema.

**Pull request merged**